### PR TITLE
Shows total memory usage

### PR DESCRIPTION
**Pull request by marcosdiez**
Now the header bar shows the total memory used by all containers

### CODE REVIEW

File: cursor.go [hunk 1]
```diff
@@ -17,6 +17,16 @@ type GridCursor struct {
 
 func (gc *GridCursor) Len() int { return len(gc.filtered) }
 
+func (gc *GridCursor) MemoryUsage() int64 {
+	var size int64
+	size = 0
+	for _, c := range gc.filtered {
+		size += c.MemUsage
+	}
+	gc.Reset()
```
Comment (bcicen) [inline cursor.go]: Why is the cursor being reset here? This breaks all forms of navigation and scrolling, forcing the cursor to the top every second.

File: cursor.go [hunk 2]
```diff
@@ -17,6 +17,16 @@ type GridCursor struct {
 
 func (gc *GridCursor) Len() int { return len(gc.filtered) }
 
+func (gc *GridCursor) MemoryUsage() int64 {
+	var size int64
+	size = 0
+	for _, c := range gc.filtered {
+		size += c.MemUsage
```
Comment (bcicen) [inline cursor.go]: Stopped, paused, etc. containers (where there is not running metric collector) have a default MemUsage value of `-1` to ensure they are omitted from displayed metrics.

On a system with 20 stopped containers, the total usage ends up showing as `-20b`

**Comment by bcicen** [comment]
Thanks for the PR -- there's a few issues with this change, please see the comments added to the diff. 

Also, there may be some confusion as to whether the "total" memory usage means all reserved/limit memory by containers or all allocated; though, this an issue that can be resolved in subsequent features/PRs.

**Comment by marcosdiez** [comment]
Hi!
I fixed all the things you suggested. Thanks for the code review.

**Comment by marcosdiez** [comment]
I thought about the issue with the meaning of "total" memory usage.
We could change the text to `combined memory usage`. That would take the double meaning away. On the other hand all I do is write a number with a unity, so I don't think no change needs to be made.
Suggestions are, as always, welcome.

**Comment by bcicen** [comment]
Great; thank you for the updates! I've merged https://github.com/bcicen/ctop/commit/712d7a5f59f06d117e8463165b32d7b75be76d2b into a feature branch for now and will merge along with some widget refactoring in the next major release.

**Pull request closed without merging**